app/screens/user_profile/custom_attributes.tsx

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {useIntl} from 'react-intl';
+import {FlatList, View, StyleSheet, type ListRenderItem} from 'react-native';
+
+import UserProfileLabel from './label';
+
+type Props = {
+    nickname?: string;
+    position?: string;
+    localTime?: string;
+    customAttributes?: DisplayCustomAttribute[];
+}
+
+const CustomAttributes = ({nickname, position, localTime, customAttributes}: Props) => {
+    const {formatMessage} = useIntl();
+
+    // Combine standard and custom attributes
+    const attributes = [
+        ...(nickname ? [{
+            id: 'nickname',
+            name: formatMessage({id: 'channel_info.nickname', defaultMessage: 'Nickname'}),
+            value: nickname,
+        }] : []),
+        ...(position ? [{
+            id: 'position',
+            name: formatMessage({id: 'channel_info.position', defaultMessage: 'Position'}),
+            value: position,
+        }] : []),
+        ...(localTime ? [{
+            id: 'local_time',
+            name: formatMessage({id: 'channel_info.local_time', defaultMessage: 'Local Time'}),
+            value: localTime,
+        }] : []),
```

Comment:
I see a pattern and this can be simplified.

You can also make the standard attributes an array of objects to be stored as a constant somewhere.  This way, if you need to expand on standard attributes, it will be a change in the constant.

```
 const standardAttributes = [
     {id: 'nickname', value: nickname},
     {id: 'position', value: position},
     {id: 'local_time', value: localTime},
 ].filter(attr => attr.value).map(attr => ({
     id: attr.id,
     name: formatMessage({
         id: `channel_info.${attr.id}`,
         defaultMessage: attr.id.charAt(0).toUpperCase() + attr.id.slice(1).replace('_', ' ')
     }),
     value: attr.value
 }));

 const attributes = [...standardAttributes, ...(customAttributes ?? [])];
```

Reply:
ah, this helps with another change I wanted to do :)

Reply:
The reason I brought this up is because often time the next person just add 3 more attributes using the same original pattern, if left the way it is.